File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--binding-to-data-from-a-service-in-a-vsto-add-in-project_2.cs
        private void AddRichTextControlAtRange()
        {
            Word.Document currentDocument = this.Application.ActiveDocument;
            currentDocument.Paragraphs[1].Range.InsertParagraphBefore();

            Document extendedDocument = Globals.Factory.GetVstoObject(currentDocument);

            richTextContentControl = extendedDocument.Controls.AddRichTextContentControl(
                currentDocument.Paragraphs[1].Range, "richTextContentControl");
            richTextContentControl.PlaceholderText =
                "Click here to download MSDN Library information about content controls.";
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--binding-to-data-from-a-service-in-a-vsto-add-in-project_2.vb
    Private Sub AddRichTextControlAtRange()

        Dim currentDocument As Word.Document = Me.Application.ActiveDocument
        currentDocument.Paragraphs(1).Range.InsertParagraphBefore()


        Dim extendedDocument As Document = Globals.Factory.GetVstoObject(currentDocument)


        richTextContentControl = extendedDocument.Controls.AddRichTextContentControl _
            (currentDocument.Paragraphs(1).Range, "richTextControl2")
        richTextContentControl.PlaceholderText = _
            "Click here to download MSDN Library information about content controls."
    End Sub